04 - Guides, Workflows, & Courses/Guides/Effective Remote Work.md

```diff
@@ -11,9 +11,9 @@ publish: true
 %% Match the title with the channel's title %% 
 
 %% Add the link (replacing #placeholder/link below) and then remove this comment%%
-Link: [YouTube](placeholder/link)
+Link: [YouTube](https://www.youtube.com/channel/UCkzyo69rqBoBJUyQ9jo53Bw)
```

Comment:
Apologies - The commit message is wrong for this commit - it should have said "Effective Remote Work"